### ISSUE TITLE
Boot support?

### ISSUE THREAD

**Issue by arichiardi** [mfikes/tach#9]
Hello Mike!

I have read a bit the code and it looks like many parts could be reusable for a `boot` task. I was wondering whether it would make sense to actually create an actual `boot` task.

I can help you with that in my spare time, I was just wondering whether you are interested or not :smile: 

Thanks for this!

**Comment by mfikes** [comment]
Yeah, my plan is to learn a bit about boot and factor out any reusable bits so a boot task can be created.
